workflowparsers/lobster/metainfo/lobster.py

```diff
@@ -79,6 +83,167 @@ class Method(runschema.method.Method):
+
+        This is only used with LOBSTER versions 4.1.0 and above.
+        """,
+    )
+
+    x_lobster_integrated_cobi_at_fermi_level = Quantity(
+        type=np.dtype(np.float32),
+        unit="joule",
+        shape=["number_of_spin_channels", "x_lobster_number_of_cobi_pairs"],
+        description="""
+        Calculated iCOBI values ingregrated up to the Fermi level.
+        """,
+    )
+
+    x_lobster_number_of_cobi_values = Quantity(
+        type=int,
+        description="""
+        Number of energy values for the COBI and iCOBI.
+        """,
+    )
+
+    x_lobster_cobi_energies = Quantity(
+        type=np.dtype(np.float32),
+        unit="joule",
+        shape=["x_lobster_number_of_cobi_values"],
+        description="""
+        Array containing the set of discrete energy values for COBI and iCOBI.
+        """,
+    )
+
+    x_lobster_cobi_values = Quantity(
+        type=np.dtype(np.float32),
+        shape=[
+            "x_lobster_number_of_cobi_pairs",
+            "number_of_spin_channels",
+            "x_lobster_number_of_cobi_values",
+        ],
+        description="""
+        Calculated COBI values.
+        """,
+    )
+
+    x_lobster_integrated_cobi_values = Quantity(
+        type=np.dtype(np.float32),
+        unit="joule",
+        shape=[
+            "x_lobster_number_of_cobi_pairs",
+            "number_of_spin_channels",
+            "x_lobster_number_of_cobi_values",
+        ],
+        description="""
+        Calculated iCOBI values.
+        """,
+    )
+
+    x_lobster_average_cobi_values = Quantity(
+        type=np.dtype(np.float32),
+        shape=["number_of_spin_channels", "x_lobster_number_of_cobi_values"],
+        description="""
+        Calculated COBI values averaged over all pairs.
+        """,
+    )
+
+    x_lobster_average_integrated_cobi_values = Quantity(
+        type=np.dtype(np.float32),
+        unit="joule",
+        shape=["number_of_spin_channels", "x_lobster_number_of_cobi_values"],
+        description="""
+        Calculated iCOBI values averaged over all pairs.
+        """,
+    )
+    x_lobster_number_of_cobi_orbital_pairs = Quantity(
+        type=int,
+        description="""
+            Number of atom orbital pairs for which are the COBIs and iCOBIs calculated.
+            """,
+    )
+    x_lobster_cobi_atom1_orbital_labels = Quantity(
+        type=str,
+        shape=['x_lobster_number_of_cobi_orbital_pairs'],
+        description="""
+                Species, indices and orbitals of the first atom for which is the specific COBI/iCOBI calculated
+                """,
+    )
+    x_lobster_cobi_atom2_orbital_labels = Quantity(
+        type=str,
+        shape=['x_lobster_number_of_cobi_orbital_pairs'],
+        description="""
+                Species and indices of the second atom for which is the specific COBI/iCOBI calculated
+                """,
+    )
+    x_lobster_cobi_orbital_values = Quantity(
+        type=np.float32,
```

Comment:
you can also use np.float64 to be more precise

Reply:
Done